### ISSUE TITLE
Publish InVEST as a Snap

### ISSUE THREAD

**Issue by mlacayoemery** [natcap/invest#121]
Hello,

It would be helpful for teaching and reproducible science to create a [Snap](https://snapcraft.io/docs/python-apps) for Linux InVEST users. This way InVEST will also be officially packaged to run on Linux with specific versions of all the required packages.

Thank you,
Martin

**Comment by phargogh** [comment]
Hey Martin, sounds interesting!  We don't have an active need for this internally, so implementing a snap build process would probably be a lower priority.  But if you're able to get a snap build process working, feel free to submit a PR, or else send us a diff or gist and we can take a look!  Otherwise, it'd probably be a while before we can take a look at this.

**Comment by mlacayoemery** [comment]
I'll add it to my fun list of things to do, but I probably won't get a closer look until later this year. It looks like getting GDAL working inside a SNAP is not going to be a snap. ;) Starting with a GDAL Docker image and adding InVEST seems pretty straight forward. I'll up date you if and when there is any progress.

**Comment by phargogh** [comment]
Sounds good!  Yes, it'll probably be much easier to get `natcap.invest` set up within a docker image than it will be to get GDAL working within a Snap.  In case it's helpful, installation of apt dependencies are documented in https://invest.readthedocs.io/en/latest/installing.html#ubuntu-debian, and `natcap.invest` should be a `pip install` away once those dependencies are installed.

**Comment by phargogh** [comment]
We've started tracking this over in the `invest-workbench` repo, so follow it there for updates!
